args.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
And this? From the documentation above, it seems to be related to tests, but shouldn't it belong to the tests directory then?

Reply:
removed